.github/workflows/test-python.yml

```diff
@@ -0,0 +1,28 @@
+name: Python Test
+
+on:
+  push:
+    branches:
+      - main
+      - renovate/**
+  pull_request:
+    branches:
+      - main
+  workflow_call:
+
+jobs:
+  test-python:
+    defaults:
+      working-directory: python
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
```

Comment:
Question: What version support do you want for the CCK. Ideally this should be a lower-bounded version of whatever the consuming cucumber flavour supports (i.e. for the CCK we support a similar amount of versions to the equivalent cucumber-ruby flavour)